thrift/src/main/thrift/atoms/video.thrift

```diff
@@ -0,0 +1,72 @@
+namespace * contentatom.video
+namespace java com.gu.contentatom.thrift.atom.video
+
+include "../shared.thrift"
+
+typedef i64 Version
+
+enum Platform {
+  YOUTUBE = 0,
+  URL = 1
+}
+
+enum Category {
+  DOCUMENTARY   = 0,
+  EXPLAINER     = 1,
+  FEATURE       = 2,
+  NEWS          = 3,
+  HOSTED        = 4, // commercial content supplied by advertiser
+  PAIDFOR       = 5
```

Comment:
Can you have a hosted documentary? Are these different things? Oh - it's not used anywhere anyway...